cobigen-cli/src/main/java/com/devonfw/cobigen/cli/utils/ParsingUtils.java

```diff
@@ -213,4 +221,30 @@ public static File getProjectRoot(File inputFile) {
         return inputFile.getAbsoluteFile().getParentFile();
     }
 
+    /**
+     * This method format the runtime generated code with the help of google API
+     * @param generatedFiles
+     *            List of generation report files
+     * @throws FormatterException
+     */
+    public static void formatJavaSources(Set<Path> generatedFiles) throws FormatterException {
+        Set<Path> format = generatedFiles;
+        Formatter formatter = new Formatter();
+        Iterator<Path> itr = format.iterator();
+        while (itr.hasNext()) {
+            try {
+                InputStream inputStream = Files.newInputStream(itr.next());
+                String unformatterContent = IOUtils.toString(inputStream);
+                String formatedCode = formatter.formatSource(unformatterContent);
+                FileOutputStream File = new FileOutputStream(inputStream.toString());
```

Comment:
In a call we already saw that this was not correctly saving the file to the right path. Please check a way to replace the old unformatted file with the formatted contents